pkg/minikube/machine/stop.go

```diff
@@ -57,6 +57,11 @@ func stop(h *host.Host) error {
 	}
 
 	if driver.NeedsShutdown(h.DriverName) {
+		klog.Infof("GuestOS: %s", h.GuestOS)
+		if h.GuestOS == "windows" {
+			// short circuit for windows and exit this branch
+			return nil
+		}
```

Comment:
now that SSH is working do we still need this?

Reply:
removed!